include/ltest-macros.lfe

```diff
@@ -1,62 +1,101 @@
+;; Include EUnit macros
 (include-lib "eunit/include/eunit.hrl")
 
+;;;===================================================================
+;;; OTP 18 hacks
+;;;===================================================================
+
+(defmacro otp-18? () `(=:= "18" (erlang:system_info 'otp_release)))
+
+(defmacro assertion-failed () `(if (otp-18?) 'assert 'assertion_failed))
+
+(defmacro assert-equal-failed ()
+  `(if (otp-18?) 'assertEqual 'assertEqual_failed))
+
+(defmacro assert-not-equal-failed ()
+  `(if (otp-18?) 'assertNotEqual 'assertNotEqual_failed))
+
+(defmacro assert-exception-failed ()
+  `(if (otp-18?) 'assertException 'assertException_failed))
+
+(defmacro assert-match-failed ()
+  `(if (otp-18?) 'assertMatch 'assertMatch_failed))
+
+
+;;;===================================================================
+;;; Helper functions
+;;;===================================================================
+
 (eval-when-compile
   (defun to-unders (atm)
     (re:replace (atom_to_list atm) "-" "_" '(#(return list) global)))
   (defun list-body
-    ((body) (when (is_list body))
-      body)
-    ((body)
-      (list body)))
+    ([body] (when (is_list body))
+     body)
+    ([body]
+     (list body)))
```

Comment:
This is a don't-be-offended-later note :-)

I never use the square brackets in my LFE projects ... so I'll eventually change this. For now it's totally cool, though -- just wanted you to know ahead of time why it will get changed ;-)